### ISSUE TITLE
R04 update following Argo User Manual / Dec 2020 review

### ISSUE THREAD

**Issue by vpaba** [nvs-vocabs/ArgoVocabs#18]
Create ID='PL', desc: 'IOPAN, Institute of Oceanology Polish Academy of Science', as reflected in the Reference Table 4 (data centres and institutions codes) updates from December 2020.

**Comment by gwemon** [comment]
@vpaba I'd recommend creating an external mapping to https://edmo.seadatanet.org/report/195 when adding this new entry.

**Comment by matdon17** [comment]
I suspect we need mappings for all the other entries as well?

**Comment by gwemon** [comment]
@matdon17 yes when you get a chance it would be good to do that. It would enable interoperability with other systems that use EDMO and it could facilitate switching to EDMO should this be the preferred option in the future. Also I was told that some Argo partners do actually use EDMO in their data system so creating the mapping as you add new entries will facilitate internal workflows/interoperability too.

**Comment by matdon17** [comment]
It'd be interesting to document examples of this and other such uses at some point to understand how it might be used/highlight to others how it can be used.

**Comment by gwemon** [comment]
@matdon17 yes it would be good indeed. I just checked back the info I received and it was Coriolis db that seems to be using EDMO instead or maybe as well as what is now in R04.
